wallet/src/server.rs

```diff
@@ -37,9 +37,9 @@ pub fn start_rest_apis(wallet_config: WalletConfig, keychain: Keychain) {
 	};
 
 	let router = router!(
-    receive_tx: get "/receive/transaction" => receive_tx_handler,
+		receive_tx: post "/receive/transaction" => receive_tx_handler,
```

Comment:
This was the first problem - it needed to be registered as a `POST` and not `GET`...